instagram/views.py

```diff
@@ -143,6 +144,28 @@ def msgreacted(request, message_id):
     conversations = message.conversation
     Reaction.objects.create(reactor=user , content_object = message, text = reaction  )
     return HttpResponseRedirect(reverse('instagram:chat', args=(conversations.id,)))
+
+def replyreact(request, reply_id):
+    reply = Reply.objects.get(pk=reply_id)
+    print(reply)
+    context = {
+        'reply': reply,
+    }
+    return render(request,'instagram/userreacts.html',context)
+
+def replyreacted(request, reply_id):
+    user = request.user
+    reply = Reply.objects.get(pk=reply_id)
+    reaction = request.POST['reaction']
+    # print(emojize(reaction))
+    # reaction = emojize(reaction)
+    conversations = reply.message.conversation
+    Reaction.objects.create(reactor=user , content_object = reply, text = reaction  )
+    return HttpResponseRedirect(reverse('instagram:chat', args=(conversations.id,)))
+
```

Comment:
Remove extra spaces from the end of file. Add a single blank line only at the end of each file